src/main/resources/languages/en/messages.yml

```diff
@@ -316,6 +316,7 @@ machines:
   HOLOGRAM_PROJECTOR:
     enter-text: '&7Please enter your desired Hologram Text into your Chat. &r(Color Codes are supported!)'
     inventory-title: 'Hologram Editor'
+    broken: '&cThe Hologram Projector you were editing has been broken, your message has been ignored.'
```

Comment:
Just noting that a new translation key is needed here

Reply:
?